### PR TITLE
Describe the release channels

### PR DESCRIPTION
**Pull request by calmh**
This is an actual PR now

### CODE REVIEW

**Comment by imsodin** [comment]
I started to review but it would be lots of redundant comments, so:
Capitalisation and quoting of release and stable in various contexts is not consistent. I would prefer always capitalizing these two words but no quoting. Thus it is clear that the concept in the scope if this documentation is meant, but it can be combined both with channel and release.

**Comment by imsodin** [comment]
Also in syncthing/syncthing#3939 "candidate releases" is used, whereas here it is "release candidates". The latter is the more common one, so I propose to keep that and switch the order in the former.

File: users/releases.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+For releases obtained from Syncthing.net or GitHub, with built-in upgrade
+functionality, the choice is made in the "Settings" dialog. Set the
+"Automatic upgrade" drop down to either "Stable releases only" or "Candidate
+releases".
+
+APT (Debian)
+~~~~~~~~~~~~
+
+The choice between release and candidate is done in the APT source
```
Comment (imsodin) [inline users/releases.rst]: between stable and candidate

File: users/releases.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+configuration. Please see `our APT instructions
+<https://apt.syncthing.net/>`__.
+
+Snap
+~~~~
+
+The ``snap`` tool can be told to install the candidate channel, but defaults
+to the release channel (``stable`` in Snap nomenclature). See the Snap
+documentation for detail.
```
Comment (imsodin) [inline users/releases.rst]: stable channel, text in parentheses obsolete.

File: users/releases.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+=========================  =========================  ======================
+**Stability**              More Stable                More Experimental
+**Features & Fixes**       About two weeks behind     Latest
+**Auto Upgrades**          Optional                   Mandatory [#]_
+**Anon. Usage Reporting**  Optional                   Mandatory
+**Support**                Fully supported            Fully supported [#]_
+=========================  =========================  ======================
+
+Run the candidate channel if you are technically savvy and enjoy new
+features. Run the stable channel if you want to minimize the amount of
```
Comment (imsodin) [inline users/releases.rst]: cursive?

File: users/releases.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+The ``snap`` tool can be told to install the candidate channel, but defaults
+to the release channel (``stable`` in Snap nomenclature). See the Snap
+documentation for detail.
+
+Some Other Distribution Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are getting packages from your Linux distribution, NAS vendor, etc.,
+then you should be getting the *release* channel. If you get a release
+*candidate* you should complain to your distributor or vendor and refer them
```
Comment (imsodin) [inline users/releases.rst]: Here it's cursive again. Probably only want cursive in the first paragraph?

File: users/releases.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+Every new feature and bugfix begins its life in the development branch,
+``master``. Every two weeks the current ``master`` becomes a *release
+candidate*. This version is identified by "-rc" in it's name, for example
+"0.14.35-rc.1".
+
+Those running the candidate channel will update to this release candidate.
+During the next twelve days it receives testing "in the wild". Any new,
+serious issues that are discovered are fixed, and new release candidates
+"0.14.35-rc.2" etc are created as needed. These release candidates do not
+include any new features or non-essential bugfixes added to "master" in the
```
Comment (imsodin) [inline users/releases.rst]: ``master``

File: users/releases.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+meantime.
+
+Once the release candidate is deemed stable, typically after twelve days, it
+becomes a *stable release* and is promoted to the stable channel. Stable
+releases are given version numbers without any suffix - "0.14.35". Unless
+any serious issues were discovered, this release is exactly identical to the
+"-rc.1" release candidated twelve days prior.
+
+The cycle then restarts two days later with a new release candidate based on
+the current "master" branch.
```
Comment (imsodin) [inline users/releases.rst]: ``master``

File: users/releases.rst [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+Those that fix a regression since the last release. Lets say the current
+release is 0.14.35. We release 0.14.36-rc.1 and discover a new problem that
+is not present in 0.14.35. This gets fixed and we release a new 0.14.36-rc.2
+candidate. However, if we discover and fix a problem that's been present
+since 0.14.20, this fix will instead be incorporated in the next regular
+cycle.
+
+What's the difference between the latest candidate and the following stable?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (imsodin) [inline users/releases.rst]: stable release

**Comment by calmh** [comment]
It's cursive

1. Once, on first mention
2. For emphasis in when they happen in the life cycle section
3. For emphasis, in that other distributors should only distribute the stable ones

File: users/releases.rst [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+Every new feature and bugfix begins its life in the development branch,
+``master``. Every two weeks the current ``master`` becomes a *release
+candidate*. This version is identified by "-rc" in it's name, for example
+"0.14.35-rc.1".
+
+Those running the candidate channel will update to this release candidate.
+During the next twelve days it receives testing "in the wild". Any new,
+serious issues that are discovered are fixed, and new release candidates
+"0.14.35-rc.2" etc are created as needed. These release candidates do not
+include any new features or non-essential bugfixes added to `master` in the
```
Comment (imsodin) [inline users/releases.rst]: Intentionally different formatting of \`master\` than in paragraph above (\`\`master\`\`)?

File: users/releases.rst [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+meantime.
+
+Once the release candidate is deemed stable, typically after twelve days, it
+becomes a *stable release* and is promoted to the stable channel. Stable
+releases are given version numbers without any suffix - "0.14.35". Unless
+any serious issues were discovered, this release is exactly identical to the
+"-rc.1" release candidated twelve days prior.
+
+The cycle then restarts two days later with a new release candidate based on
+the current `master` branch.
```
Comment (imsodin) [inline users/releases.rst]: dito last comment.

**Comment by calmh** [comment]
@imsodin thanks for the sharp eyes! I'm merging this now, but please feel free to fix anything else you find that's bad.

**Comment by calmh** [comment]
@st-review merge it

**Comment by st-review** [comment]
@calmh: Build status is `pending`. I'll wait until it goes green and then merge!

**Comment by st-review** [comment]
:ok_hand: Merged as 6f7ca2efd59abeb78053d9a95f65b208f89b8a31. Thanks, @calmh!

**Pull request closed without merging**